Source/EasyNetQ.Management.Client/DeleteQueueCriteria.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace EasyNetQ.Management.Client;
+
+public record DeleteQueueCriteria(bool _IfUnused, bool _IfEmpty)
```

Comment:
@Pliner I had a conflict between `property bool IfUnused` from ctor and `static DeleteQueueCriteria IfUnused`.

I preferred to keep the latter one, because no one should really create new instances of DeleteQueueCriteria or access their properties.

Prefixing with _ is bad, I'll take any renaming suggestion you have :)

Reply:
@Pliner why your review is on hold? :)

Reply:
@Pliner ping :)

I remind you, that currently master branch is "broken" by 40d45ab

Reply:
maybe just ifUnused and ifEmpty?

Reply:
Renamed to ifUnused/ifEmpty